arch/arm64/configs/bcm2712_defconfig

```diff
@@ -1536,6 +1536,7 @@ CONFIG_INV_MPU6050_I2C=m
 CONFIG_APDS9960=m
 CONFIG_AS73211=m
 CONFIG_BH1750=m
+CONFIG_TCS3472=m
 CONFIG_TSL4531=m
 CONFIG_VEML6070=m
```

Comment:
I think `CONFIG_VEML6040=m` needs to be added to the defconfigs.

Reply:
Indeed - it's been missing from the start, but not tested. It's there now, and there will be another test build in about 40 minutes.